### PR TITLE
Update package.json

### PR DESCRIPTION
**Pull request by JS31096**
Updating xmldom package version

### CODE REVIEW

**Comment by larrymyers** [comment]
@JS31096 for this change to be available on npm the minor version in the package.json must be bumped.

**Comment by JS31096** [comment]
Hi @larrymyers ,

so, will there be any planned minor releases in the future for the changes to be available.

"xmldom": "^0.1.22" is throwing dependabot alerts so made the change.

Please let me know if there can be any release so that I can contribute accordingly and update minor version release to 2.3.3 in package.json

Thanks!

File: package.json [hunk 1]
```diff
@@ -2,7 +2,7 @@
   "author": "Larry Myers",
   "name": "jasmine-reporters",
   "description": "Reporters for the Jasmine BDD Framework",
-  "version": "2.3.2",
+  "version": "2.3.3",
```
Comment (bduffany) [inline package.json:5]: This is the patch version, I think @larrymyers  wants the minor version to be bumped instead?

```suggestion
  "version": "2.4.0",
```

**Comment by Arjan321** [comment]
Hi @larrymyers , is there a possibility for this to be merged and released? This is currently causing [`npm audit` failures](https://npmjs.com/advisories/1650).

**Comment by bduffany** [comment]
Friendly ping, we have a transitive dep on this package and it's causing a security warning in our repo. Would be great to get this merged in. Anything I can do to help with this? For example, if you're waiting to merge until there's more extensive testing to make sure that this works, I'd be happy to help with that, but would need some pointers on how to go about doing so.

**Comment by putermancer** [comment]
No, I think it's mostly that this package is in extreme maintenance mode, and neither of us spends much time on this project these days. So it's a combination of "out of sight, out of mind," and "the squeaky gets the grease."

Thanks for squeaking, and thanks for the update. Merging!

**Comment by putermancer** [comment]
Merged, tagged, published to npm. Have a good day!

**Comment by bduffany** [comment]
Thanks!

**Pull request merged**